algoritm/recursive/factorial.js

```diff
@@ -0,0 +1,17 @@
+const readline = require("readline");
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+rl.on("line", (N) => {
+  N = parseInt(N);
+  console.log(factorial(N));
+  rl.close();
+}).on("close", () => {
+  process.exit();
+});
```

Comment:
백준 문제 푸실 때 이렇게 한 줄로도 입력 받을 수 있어요 :)

```suggestion
const N = parseInt(require("fs").readFileSync("/dev/stdin").toString());
```

Reply:
fs랑 readline중에서 고민하다가 지금까지 계속 readline만 써와서 이번에도 readline썼는데 다음엔 fs 사용해볼게요!